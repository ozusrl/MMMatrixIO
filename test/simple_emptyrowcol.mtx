%%MatrixMarket matrix coordinate real general
5 5 5
1 1 10.1
2 1 10.4
2 3 10.5
4 1 10.7
4 3 10.9
