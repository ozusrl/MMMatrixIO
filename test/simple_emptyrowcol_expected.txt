############### MM  ##############
5 5 5
0 0 10.1
1 0 10.4
1 2 10.5
3 0 10.7
3 2 10.9
############### COO ##############
int rowIndices[5] = {
0,
1,
1,
3,
3,
};
int colIndices[5] = {
0,
0,
2,
0,
2,
};
double values[5] = {
10.1,
10.4,
10.5,
10.7,
10.9,
};
############### CSR ##############
int rowPtr[6] = {
0,
1,
3,
3,
5,
5,
};
int colIndices[5] = {
0,
0,
2,
0,
2,
};
double values[5] = {
10.1,
10.4,
10.5,
10.7,
10.9,
};
############### CSC ##############
int rowIndices[5] = {
0,
1,
3,
1,
3,
};
int colPtr[6] = {
0,
3,
3,
5,
5,
5,
};
double values[5] = {
10.1,
10.4,
10.7,
10.5,
10.9,
};
